syntaxes/omt.tmLanguage.json

```diff
@@ -343,7 +343,7 @@
                 }
             ]
         },
-        "_notification-action-def": {
+        "_system-message-action-def": {
```

Comment:
The system message action had different properties from the notification action, so these need to be altered here.